### ISSUE TITLE
UnnecessarySafeNavigationOperator Rule false positive: spread operator

### ISSUE THREAD

**Issue by tosts** [CodeNarc/CodeNarc#90]
After updating to 0.23 from 0.21 and adding the new rule we see violations in the form of:
```
[SRC]def (timeA, timeB) = [a, b]*.timeLast
[MSG]The safe navigation operator (?.) is unnecessary for "[a, b]" in class com.one.of.our.Classes
```
This is most likely not the intended behavior of the rule.

**Comment by chrismair** [comment]
Looks like Groovy added a new *spreadSafe* property on expressions, that helps distinguish between multiple assignments and explicit use of the safe navigation (?.) operator.

Fixed for CodeNarc 0.24.

**Comment by tosts** [comment]
Late, but thanks. :)
